pkg/aaparser/aaparser.go

```diff
@@ -26,7 +25,7 @@ func GetVersion() (int, error) {
 // LoadProfile runs `apparmor_parser -r` on a specified apparmor profile to
 // replace the profile.
 func LoadProfile(profilePath string) error {
-	_, err := cmd("", "-r", filepath.Dir(profilePath))
```

Comment:
I think this line is equivalent to running command `apparmor_parser -r /tmp`, if I didn't miss anything. And `apparmor_parser -r /tmp` dose not work fine.

FIY: I came across this issue, when I imported this package in other project.